vendor/bat-native-ledger/src/bat/ledger/internal/uphold/uphold_card.cc

```diff
@@ -42,6 +42,12 @@ UpholdCard::~UpholdCard() {
 void UpholdCard::CreateIfNecessary(
     ledger::ExternalWalletPtr wallet,
     CreateCardCallback callback) {
+  if (!wallet) {
```

Comment:
If `wallet` was a `nullptr` before this defensive code was added, could it have caused any side effects other than a crash?